server/lib/src/constants/schema.rs

```diff
@@ -211,6 +211,40 @@ pub const JSON_SCHEMA_ATTR_NAME_HISTORY: &str = r#"{
     }
 }"#;
 
+pub const JSON_SCHEMA_ATTR_EC_KEY: &str = r#"{
+    "attrs": {
+      "class": [
+        "object",
+        "system",
+        "attributetype"
+      ],
+      "description": [
+        "Account verification public and optionally private key."
```

Comment:
I think we should actually just make this a private key only. I think public will have to eventually be a seperate type to allow for splitting the values and ensuring at a type/attr level we don't leak across trust boundaries. 

How angy will you be if I ask you to make this "EC_KEY_PRIVATE" rather than EC_KEY?

Reply:
very mildly :p, but actually makes sense